purchases/src/test/java/com/revenuecat/purchases/responses.kt

```diff
@@ -1,23 +0,0 @@
-package com.revenuecat.purchases
-
-import java.io.File
-
-class Responses {
-    companion object {
-        val validFullPurchaserResponse by lazy {
-            getJSONFromPath("responses/valid_full_purchaser_response.json")
-        }
-        val validEmptyPurchaserResponse by lazy {
-            getJSONFromPath("responses/valid_empty_purchaser_response.json")
-        }
-        val subscriberAttributesErrorsPostReceiptResponse by lazy {
-            getJSONFromPath("responses/attribute_errors_post_receipt_response.json")
-        }
-
-        private fun getJSONFromPath(fileName: String): String {
-            val classLoader = this::class.java.classLoader
-            val resource = classLoader!!.getResource(fileName)
-            return File(resource.path).readText()
-        }
-    }
-}
```

Comment:
So I couldn't get sharing test resources from one module across different modules to work and I decided to not spend much time on it and removed the files and added the strings here. The new file is here https://github.com/RevenueCat/purchases-android/pull/180/files#diff-98a29a61b1056eb7d9f86e154f51323c